sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -235,6 +251,12 @@
    */
   private StateKey.BagUserState cachedPartialBagUserStateKey;
 
+  /**
+   * This member should only be accessed indirectly by calling
+   * {@link #createOrUseCachedMultimapSideInputKey} and is only valid during {@link #processElement}
+   * and is null otherwise.
+   */
+  private StateKey.MultimapSideInput cachedPartialMultimapSideInputKey;
```

Comment:
<!--new_thread; commit:d1c0ffde59d0b17126deecb840cbeec36719d7a6; resolved:0-->
@Nullable, and on other fields described to be null

Reply:
Done